src/RegionSelect.js

```diff
@@ -0,0 +1,22 @@
+import Select from 'react-select'
+import { customStyles } from './selectStyling';
+
+function RegionSelect({ data, regionSelect }) {
+
+    const regionData = data;
+    const regionNames = regionData.map((network, index) => ({ value: `${network.id}`, label: `${network.name}` }));
+
+    const handleSelect = (selected) => {
+        regionSelect(selected.value)
```

Comment:
small thing but perhaps you could combine handle select with regionSelect? since its basically one line of code. also it's not clear when choosing a region what exactly the names you are looking at represent. i thought i was looking at city names at first until i saw there were a bunch of Onroll options

Reply:
Added City next to bike share company